### ISSUE TITLE
Mini FTool - Main window needs to be active for hotkey press?

### ISSUE THREAD

**Issue by DoMx** [ils94/PyFlyff#3]
First up, LOVE the tool. As a desktop player grinding on a second monitor while I work etc, this is huge for me.

I'm playing like this:
Main Window (with menu) - RM
Second window - DPS

So I've set up my FTool so my RM heals when I press the hotkey. BUT, this only works when I have the Main window selected. Is this the intended functionality? It would be great it if I could be playing my DPS on my second window and hit hotkey to heal from my RM, if that makes sense?

Also, if you have a non-crypto donation link, I'd love to support.

**Comment by ils94** [comment]
I see, in that case it working as intended.

The Mini Ftool is only functional in the Main Window, the hotkey to activate the loop is, in fact, from the Main Window.

And thats why I made the "Alt Control" which is exactly what you are trying to do.

Pick a hotkey for the Main Client to send the command to the Alt Client, so:

Main Client Hotkey: F2 / E / Q (you pick)

Alt Client Hotkey: F1, F3, F4... (note that both Main Client and Alt Client hotkeys most be different (this is a limitation I still didn't figure out)

Then when you press "Start", once you hit for example E on Main Client, it will send a command to press "F1 or whatever key" you set for the Alt Client.

Basically you can command your FS/RM to heal you while you still playing on the Main Client with your Main Character, removing completly the need of Alt + Tabbing and of course, making it so you dont have to use Ftool and get yourself in trouble for using botting tools.
